--- a/node_modules/frida-java-bridge/lib/class-model.js
+++ b/node_modules/frida-java-bridge/lib/class-model.js
@@ -282,7 +282,8 @@ model_new (jclass class_handle,
       get_method_name (jvmti, method, &name, NULL, NULL);
       get_method_modifiers (jvmti, method, &modifiers);
 
-      model_add_method (model, name, method, modifiers);
+      if (name[0] != '<')
+        model_add_method (model, name, method, modifiers);
 
       deallocate (jvmti, name);
     }
